apiclient/src/ds_api/grpc.rs

```diff
@@ -283,15 +276,20 @@ impl DsGrpcClient {
         group_id: GroupId,
         epoch: GroupEpoch,
         group_state_ear_key: &GroupStateEarKey,
-        signing_key: &PseudonymousCredentialSigningKey,
+        signing_key: &ClientSigningKey,
     ) -> Result<WelcomeInfoIn, DsRequestError> {
         let qgid: QualifiedGroupId = group_id.try_into()?;
+
+        let signature_public_key =
+            SignaturePublicKey::from(signing_key.credential().verifying_key().clone());
```

Comment:
Didn't you want to use the `ClientVerifyingKey` here?

Reply:
c61ab84